### PR TITLE
Feature/file title

### PR DESCRIPTION
**Pull request by iglimanaj**
In my application I had the need to set custom filenames for the documents, otherwise all documents loaded by vue-pdf-app would be downloaded as `document.pdf`.  I could solve this issue in my app by simply calling the following function in the `openHandler`:

`PDFViewerApplication.setTitleUsingUrl(this.titttle)`

Perhaps this could be an interesting feature for the community, therefore I introduced in this pull request a new property, called `fileTitle`.

### CODE REVIEW

**Review by sandanat — COMMENTED** [review]
Please check review comments

File: src/views/pdf-app.vue [hunk 1]
```diff
@@ -2,6 +2,7 @@
   <pdf-viewer
     :pdf="pdf"
     :config="config"
+    downloadFileName="test"
```
Comment (iglimanaj) [inline src/views/pdf-app.vue:5]: @sandanat you can check once again with this example

File: src/components/pdf-viewer.vue [hunk 2]
```diff
@@ -1005,6 +1005,9 @@ export default class PdfViewer extends Vue {
   @Prop({ required: false, type: Boolean, default: () => false })
   private title!: boolean;
 
+  @Prop({ required: false, type: String })
+  private fileTitle!: string;
```
[the remark this answers is not in the document]
Comment (iglimanaj) [reply src/components/pdf-viewer.vue]: Yep, I used ? in the newest commit and changed the property name to `downloadFileName`

[the remark this answers is not in the document]
Comment (sandanat) [reply src/components/pdf-viewer.vue]: Change prop to `fileName` please

[the remark this answers is not in the document]
Comment (iglimanaj) [reply src/components/pdf-viewer.vue]: @sandanat done :+1:

**Review by sandanat — APPROVED** [review]

**Pull request merged**